cves/CVE-2014-9604.yml

```diff
@@ -346,7 +363,7 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: The developer responsible for this vulnerability made the mistake of trusting user input.
```

Comment:
Please elaborate. Did they violate specification? What logical mistake did the developer make when writing this file? Get into the mind of the developer.